django/db/models/expressions.py

```diff
@@ -150,12 +150,6 @@ def __init__(self, output_field=None):
         if output_field is not None:
             self.output_field = output_field
 
-    def __getstate__(self):
-        # This method required only for Python 3.4.
-        state = self.__dict__.copy()
-        state.pop('convert_value', None)
```

Comment:
@timgraham looks like this broke expression pickling:
```
Traceback (most recent call last):
  File "/home/travis/build/millerdev/django-cte/tests/test_recursive.py", line 212, in test_pickle_recursive_cte_queryset
    pickled_qs = pickle.loads(pickle.dumps(regions))
AttributeError: Can't pickle local object 'BaseExpression.convert_value.<locals>.<lambda>'
```
So apparently this is required for more than Python 3.4 (I'm seeing it on Python 3.5 and 3.6). This was originally added by https://github.com/django/django/pull/8854.

Found here: https://travis-ci.org/millerdev/django-cte/jobs/366128465